exist-core/src/test/java/org/exist/xquery/XQueryContextTest.java

```diff
@@ -27,18 +27,18 @@
 import org.junit.Test;
 import org.easymock.EasyMock;
 
+import javax.xml.XMLConstants;
 import java.io.IOException;
 import java.lang.reflect.Field;
-import java.util.Deque;
-import java.util.List;
+import java.util.*;
 
 import static org.easymock.EasyMock.*;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
 
-/**
- * @author <a href="mailto:adam.retter@googlemail.com">Adam Retter</a>
- */
 public class XQueryContextTest {
+    private static final List<String> INITIAL_NAMESPACES = Arrays.asList(
```

Comment:
1. Rather than encoding strings here, can these be references to constants that are already defined elsewhere in eXist-db perhaps?
2. As it appears that these are only ever used in a `HashSet`, why not just declare a `HashSet` here instead of a `List` whose member references are later copied to a `HashSet` at each site of use?
3. As this is for expected values for an assertion, this list should ideally be made immutable, i.e. `Set.of(...)`
3.

Reply:
If this list is modified by even a single test, then it should not be global shared state in the test class, otherwise you have potential side-effects between tests.